package.json

```diff
@@ -3,7 +3,6 @@
   "description": "",
   "version": "1.0.0",
   "main": "app.js",
-  "devDependencies": {},
```

Comment:
Was the removal of `devDependencies` automatic? This should still be here. If it's empty it's the same I guess but still weird and an unnecessary removal.

Reply:
Yes I didn't make any manual changes to package.json.

Maybe I added some unnecessary libraries like `axio` when doing` node install`. Something that must be checked